umbraco/Umbraco-CMS
# V8/feature/super doctype create menu

### Prerequisites

- [^ ] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes https://github.com/umbraco/Umbraco-CMS/issues/6410

### Description

Hi, this is an update to the Document Type Create Menu.

Why? - there is a blog post here that contains most of the thinking: https://moriyama.co.uk/about-us/news/creating-doctypes-in-umbraco-v8/

I've been using it as a 'getting started' training support tool when working with people who are new to Umbraco to help them get used to the concepts. I released it as a package:

https://our.umbraco.com/packages/backoffice-extensions/super-doc-type-create-menu/

There has been a clamour to add it to the core, but as the issue, this is linked to states - HQ have better plans than this for the Doc Type menu.

However, have been asked if I would submit a PR anyway by thechiefunicorn:

![image](https://user-images.githubusercontent.com/260802/96174681-900c6180-0f21-11eb-9aac-368a346c1510.png)

and so I am doing so.

### To test this

Go to Settings - Document Types - and open the Create Menu, you should now see an updated menu with different icons, and descriptions for different types of Document Types you can create, the thinking is in the blog post above.

It's worth testing each menu item, to see if it performs as advertised, the Element Type option should create a Document Type with the 'Is Element Type' checkbox already checked.

### Wording

In terms of the wording of the options, I'm sure it could be improved, really it's the icon and the clear label of the option, and the order of the options that is the improvement (eg no Document Type Without Template option). But I'm happy if people maybe want to tweak the descriptions, etc

I'm aware this removes the Document Type Collection option.

That's it I think!

Thanks for considering this contribution to Umbraco CMS!
